mxcube3/ui/components/MXNavbar/MXNavbar.css

```diff
@@ -1,3 +1,8 @@
+.navbar-fixed-top {
```

Comment:
Hm, this does not seem to be the fix that I had in mind, this makes the entire "nav" bar floating, which I agree might be useful . But the original problem is that the sample grid "control" bar used to be floating, which I think is even more important. Either both or only the sample grid "control" bar should be floating.

The problem seems to be that the "nav" bar hides the sample grid "control" bar when its floating, applying an offset when floating might work ...

Reply:
I would say both should be always visible, I do not know about the 'control' bar, but the nav bar was floating before so thats what I tried to fix. I know it is not the "best" fix, but it is working as intended